synapse/handlers/federation_event.py

```diff
@@ -167,8 +169,8 @@ def __init__(self, hs: "HomeServer"):
 
         self._send_events = ReplicationFederationSendEventsRestServlet.make_client(hs)
         if hs.config.worker.worker_app:
-            self._user_device_resync = (
-                ReplicationUserDevicesResyncRestServlet.make_client(hs)
+            self._multi_user_device_resync = (
+                ReplicationMultiUserDevicesResyncRestServlet.make_client(hs)
```

Comment:
I was going to suggest renaming all of these things to remove the `multi`, that can be done as a follow-up though.